### PR TITLE
Wdp200103 3

### PR DESCRIPTION
**Pull request by mariPab**
Feature boxes are now links rather than divs, components are styled on hover, outer border of icon changes its color on hover

### CODE REVIEW

**Comment by Varenthein** [comment]
And where's the descirption?

File: src/components/common/FeatureBox/FeatureBox.module.scss [hunk 1]
```diff
@@ -61,10 +62,14 @@
     }
   }
 
-  &.active {
+  &:hover {
     .iconWrapper {
       color: #ffffff;
```
Comment (Varenthein) [inline src/components/common/FeatureBox/FeatureBox.module.scss]: Please strore colors in sass variables.

Comment (mariPab) [reply src/components/common/FeatureBox/FeatureBox.module.scss]: All colors in FeatureBox.module.scss are now stored in variables

**Comment by Varenthein** [comment]
And waht about it? ;)

image

**Comment by mariPab** [comment]
Moved to Wdp200103 3 v2

**Pull request closed without merging**